VEHICLE/GetVehicleWheelType.md

```diff
@@ -8,25 +8,28 @@ ns: VEHICLE
 int GET_VEHICLE_WHEEL_TYPE(Vehicle vehicle);
 ```
 
-```
-Returns an int  
-Wheel Types:  
-0: Sport  
-1: Muscle  
-2: Lowrider  
-3: SUV  
-4: Offroad  
-5: Tuner  
-6: Bike Wheels  
-7: High End
-8: Benny's Original
-9: Benny's Bespoke
-10: Open Wheel
-11: Street
-Tested in Los Santos Customs  
+```c
+Returns an int
```

Comment:
It would probably be a good idea to move this to the Return value section

Reply:
Maybe it would be better to just remove that completely?

Reply:
Either works, but it would still be g ood to tell them whats returned, say `Returns an integer of eVehicleWheelType`.

Reply:
Not 100% where to do that, would you mind by any chance making a pull request for my fork with that?

Reply:
## Return value
```
Returns an integer of eVehicleWheelType
```

Like this you mean?

Reply:
This has been done